File: 백준/Bronze/2903. 중앙 이동 알고리즘/중앙 이동 알고리즘.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		sc.close();
		
		// 2 3 5 9
		int line = (int)Math.pow(2, n)+1;
		System.out.println((int)Math.pow(line,2));

	}

}

File: 백준/Bronze/2903. 중앙 이동 알고리즘/중앙 이동 알고리즘.cc
#include <iostream>
#include <cmath>
using namespace std;

int main(){
    int N;
    cin >> N;
    
    // 2 3 5 9
    int line = pow(2, N)+1;
    cout << (int)pow(line,2);
    
    return 0;
}